src/assets/mapStyles/mapStyles.js

```diff
@@ -4,16 +4,24 @@ export default {
         sources: {
             basemap: {
                 type: "vector",
-                url: "http://localhost:8086/data/basemap.json"
+                // This URL will pull the basemap GeoJson from S3 so that no local tile server is required.
+                // NOTE: this location is under development and may change. If you suddenly lose parts of the map, it
+                // may be that we have moved the tiles to a new location on S3.
+                "tiles": ["https://d38anyyapxci3p.cloudfront.net/baseTiles_3/{z}/{x}/{y}.pbf"]
```

Comment:
@wdwatkins & @abriggs-usgs can we converge yet on the whole hru data goes in the /xxx dir.. and the basetiles go in the /xxx dir instead? that way we don't have to continue manually moving dirs and tiles around? 

the jenkins job seems to push out the non-data-joined tiles to https://github.com/usgs-makerspace/wbeep-processing/blob/master/jenkins/tippecanoe_create_tiles_Jenkinsfile#L30, would we eventually have the tiles joined with data in the same place? or could we decide now and stop changing these :)?

Reply:
This PR is really a proof of concept. There are still some kinks to work out. But, I thought we may as well try it out across the group, and see how it works.

Reply:
This looks good to me.  As far as a definitive location for the tiles my vote is for tiles and basetiles in a given bucket, but it's pretty easy for me to change in the automation stuff I'm doing if we need to in the future.